app/src/hid.c

```diff
@@ -15,6 +15,8 @@ static struct zmk_hid_keyboard_report keyboard_report = {
 
 static struct zmk_hid_consumer_report consumer_report = {.report_id = 2, .body = {.keys = {0}}};
 
+static struct zmk_hid_plover_report plover_report = {.report_id = 0x50, .body = {.buttons = {0}}};
```

Comment:
If you're hardcoding this report ID, please make it a constant w/ detail/comments. I understand host limitations resulted in this funkiness.

Reply:
Sure, I'll also try to get an issue on the hidapi-repo so that the funkiness won't be needed in the future. But the main reason is that we can't get the report descriptor from the device in a platform independent way, so we won't know which report id matches our usage page.

That could probably be solved by adding hidapi-support for finding out which usage page maps to which report id, which I believe can be done on all operating systems.